### ISSUE TITLE
External Player only checks for Flash

### ISSUE THREAD

**Issue by issuemover631** [airsonic/airsonic#380]
<a href="https://github.com/Lightsockie"><img src="https://avatars0.githubusercontent.com/u/1125399?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Lightsockie](https://github.com/Lightsockie)**
_Thursday May 04, 2017 at 03:32 GMT_
_Originally opened as https://github.com/Libresonic/libresonic/issues/379_

----

From some discussion in the IRC, it seems the external player created from share links only checks for flash, bypassing jwplayer completely.

As seen [here](https://github.com/Libresonic/libresonic/blob/4d365f4024ca9548470357174c36b1c842d8c091/libresonic-main/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp#L7):
`/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp`
```html
<%--@elvariable id="model" type="java.util.Map"--%>
<%@ page language="java" contentType="text/html; charset=utf-8" pageEncoding="iso-8859-1" %>

<html>
<head>
    <%@ include file="head.jsp" %>
    <script type="text/javascript" src="<c:url value="/script/swfobject.js"/>"></script>
    <script type="text/javascript" src="<c:url value="/script/prototype.js"/>"></script>

```

**Comment by jooola** [comment]
cc @airsonic/maintainers What the state of our current html5 player ? Does the external player also have an html5 player ?

**Comment by muff1nman** [comment]
See #448 for the todo to migrate the external player to MediaElement. Once that is done, this can be resolved.

**Comment by jvoisin** [comment]
This was fixed by #962
